PL2/PL2-core/pom.xml

```diff
@@ -26,6 +26,11 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-all</artifactId>
+			<version>1.9.5</version>
+		</dependency>
+        <dependency>
```

Comment:
We will use the current highest release, which we will keep for the entire project unless some future release is required for certain functionality.

currently, the highest release version is `1.10.19`, so we will use that (see [Maven Repos](http://mvnrepository.com/artifact/org.mockito/mockito-all)